### DISCUSSION TITLE
App Icon

### DISCUSSION THREAD

**Discussion by shuvashish76** [iamr0s/Dhizuku#55] [Ideas]
Hello, what type of icons should be used for the app? Any specific type? @iamr0s

**Comment by nktnet1** [comment]
Related to #61.

Looks like this can be closed - resolved by #129.

<img width="925" height="883" alt="pr-129-dhizuku-icon" src="https://github.com/user-attachments/assets/06be6ac0-8b34-4e30-b8f0-60a6ef110809" />
